src/generator/generator.ts

```diff
@@ -7,40 +7,45 @@ import { serialize } from '@azure-tools/codegen';
 import { Host, startSession } from '@azure-tools/autorest-extension-base';
 import { codeModelSchema, CodeModel } from '@azure-tools/codemodel';
 import { values } from '@azure-tools/linq';
-import { generateClient } from './client';
-import { generateModels } from './models';
 import { generateOperations } from './operations';
+import { generateModels } from './models';
 import { generateEnums } from './enums';
+import { generateClient } from './client';
+import { generateTimeHelpers } from './time';
 
 // The generator emits Go source code files to disk.
-export async function convenienceGen(host: Host) {
+export async function protocolGen(host: Host) {
   const debug = await host.GetValue('debug') || false;
 
   try {
 
     // get the code model from the core
     const session = await startSession<CodeModel>(host, codeModelSchema);
+    const operations = await generateOperations(session);
 
     // output the model to the pipeline.  this must happen after all model
     // updates are complete and before any source files are written.
     host.WriteFile('code-model-v4.yaml', serialize(session.model), undefined, 'code-model-v4');
 
-    const client = await generateClient(session);
-    host.WriteFile('client.go', client, undefined, 'source-file-go');
-
-    const operations = await generateOperations(session);
     for (const op of values(operations)) {
       host.WriteFile(`${op.name.toLowerCase()}.go`, op.content, undefined, 'source-file-go');
     }
 
     const enums = await generateEnums(session);
     if (enums.length > 0) {
-      host.WriteFile('enums.go', enums, undefined, 'source-file-go');
+      host.WriteFile(`enums.go`, enums, undefined, 'source-file-go');
```

Comment:
Why did you change to these types of quotes? According to me the correct thing is to use ' ' if it's a simple string as above.

Reply:
It was from copy/paste from the other layer, will fix.